CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
+1. Fork the repository.
+1. Create an `upstream` remote.
+```bash
+$ git remote add upstream git@github.com:fastly/terraform-provider-fastly.git
+```
```

Comment:
Doesn't this usually happen automatically, and it calls the remote 'origin'?

Reply:
@triblondon if you were to fork the repository, then your fork would be `origin` and there would be nothing "pointing back" to github.com/fastly/terraform-provider-fastly.

[Here](https://docs.github.com/en/enterprise/2.20/user/github/getting-started-with-github/fork-a-repo#keep-your-fork-synced) is the GitHub documentation, specifically step 7.

Reply:
🤦 I saw 'fork' and imagined 'clone'.  Ignore me completely.

Reply:
No worries! It's easy to overlook :)